build/PlatformInstaller.targets

```diff
@@ -0,0 +1,300 @@
+<Project>
+  <Target Name="BuildInstallers" DependsOnTargets="GenerateTargzs;GenerateRpms;GenerateDebs;GenerateRelabledInstallers" />
+
+  <Target Name="_EnsureInstallerPrerequisites">
+    <MakeDir Directories="$(_InstallersOutputDir)" />
+
+    <!-- Check Docker server OS -->
+    <Exec Command="docker version -f &quot;{{.Server.Os}}&quot;" StandardOutputImportance="Normal" ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="DockerHostOS" />
+    </Exec>
+
+    <Error Text="Docker host must be using Linux containers." Condition="'$(DockerHostOS)' != 'linux'"/>
+    <Error Text="Linux archive not found at $(SFLinuxArchiveFilePath)." Condition="!Exists('$(SFLinuxArchiveFilePath)')" />
+  </Target>
+
+  <Target Name="_DownloadInstallers">
+    <!-- Download dotnet installers -->
+    <MakeDir Directories="$(_InstallerSource)" />
+    <!--
+      Note: KOREBUILD_DOTNET_FEED_CREDENTIAL is intentionally NOT an MSBuild variable.
+      MSBuild doesn't to the substitution correctly because the string contains %,
+      so we'll let bash do it instead.
+    -->
+    <Exec Command="curl --fail -sSL &quot;$(RuntimeTargzLink)$KOREBUILD_DOTNET_FEED_CREDENTIAL&quot; -o $(_InstallerSource)$(DotnetRuntimeInstallerArchiveName)" />
```

Comment:
Use DotNetAssetRootAccessTokenSuffix instead of KOREBUILD_DOTNET_FEED_CREDENTIAL. 

Would be better to switch to the [`DownloadFile`](https://github.com/aspnet/BuildTools/blob/dev/modules/KoreBuild.Tasks/DownloadFile.cs) task now. This has some additional protections, like timeouts, and support for `file://` URIs.